frameworks/FSharp/giraffe/giraffe-jsonutf8.dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM microsoft/dotnet:2.1-sdk-stretch AS build
```

Comment:
need to rename this docker file to match name change 
`giraffe-jsonutf8.dockerfile` -> `giraffe-utf8json.dockerfile`

Reply:
oops